tests/System.Text.Utf8String.Tests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
For shipping libraries, it is necessary to get versioning, description, etc. However, it is not really necessary in this particular assembly since we aren't doing any friend assembly stuff. I'll remove it. We can always add one later if we need it.